interp/cfg.go

```diff
@@ -877,6 +898,12 @@ func (interp *Interpreter) cfg(root *node, sc *scope, importPath, pkgName string
 			gotoLabel(n.sym)
 
 		case callExpr:
+			for _, c := range n.child {
+				if isBlank(c) {
+					err = n.cfgErrorf("cannot use _ as value")
+					return
```

Comment:
I know you used a return because a break would only get us out of L906, but now I'm wondering if we should not change all the breaks (like L785) in returns (even though it's not necessary) to show the same intent everywhere.
But it can be in another PR.